R/beqs.R

```diff
@@ -0,0 +1,49 @@
+
+##
+##  Copyright (C) 2015  Whit Armstrong and Dirk Eddelbuettel and John Laing
+##
+##  This file is part of Rblpapi
+##
+##  Rblpapi is free software: you can redistribute it and/or modify
+##  it under the terms of the GNU General Public License as published by
+##  the Free Software Foundation, either version 2 of the License, or
+##  (at your option) any later version.
+##
+##  Rblpapi is distributed in the hope that it will be useful,
+##  but WITHOUT ANY WARRANTY; without even the implied warranty of
+##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+##  GNU General Public License for more details.
+##
+##  You should have received a copy of the GNU General Public License
+##  along with Rblpapi.  If not, see <http://www.gnu.org/licenses/>.
+
+
+##' This function uses the Bloomberg API to retrieve 'beqs' (Bloomberg
+##' EQS Data) queries
+##'
+##' @title Run 'Bloomberg EQS' Queries
+##' @param screenName A character string with the name of the screen to execute.
+##'  It can be a user defined EQS screen or one of the Bloomberg Example screens on EQS
+##' @param screenType A character string of value PRIVATE or GLOBAL
+##' Use PRIVATE for user-defined EQS screen.
+##' Use GLOBAL for Bloomberg EQS screen.
+##' @param Group An optional character string with the Screen folder name as defined in EQS
+##' @param PiTDate A character string with the Point in Time Date of the screen to execute.

```

Comment:
Could 'Group' and 'PitDate' be lower-case arguments (which is more common) or does Bloomberg use uppercase?